cmd/gpu-feature-discovery/main.go

```diff
@@ -86,6 +86,13 @@ func main() {
 			Value:   "/etc/kubernetes/node-feature-discovery/features.d/gfd",
 			EnvVars: []string{"GFD_OUTPUT_FILE"},
 		},
+		&cli.StringFlag{
+			Name:    "imex-nopdes-config",
+			Aliases: []string{"imex-nodes-config"},
```

Comment:
You don't need this alias with the correct spelling in the `Name` field

Reply:
done